src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -899,9 +899,12 @@
         single id, <emphasis>anyone</emphasis>, that represents
         anyone.</para></listitem>
 
-        <listitem><para><emphasis role="bold">auth</emphasis> doesn't
-        use any id, represents any authenticated
-        user.</para></listitem>
+        <listitem><para><emphasis role="bold">auth</emphasis> is a convenience
+        scheme which defaults to the currently-authenticated user and scheme.
+        Any ID which is provided using this scheme is ignored by ZooKeeper.
```

Comment:
Thanks for the patch, it is a good improvement on doc.

I think the ID here refers to the id of the scheme:id pair of the ID object in the ACL, correct? Mentioning this because the auth scheme is also referenced in command line where people can do 'setAcl /node auth:username:password:crdwa' in which case the username (sometimes overloaded as id) is required.